xbmc/music/MusicDatabase.h

```diff
@@ -140,6 +140,7 @@ class CMusicDatabase : public CDatabase
    \param rating [in] a rating for the song
    \param userrating [in] a userrating (my rating) for the song
    \param votes [in] a vote counter for the song rating
+   \param strSongvideoURL [in] url to video of the song
```

Comment:
`strSongvideoURL` -> `songvideoURL` to match the actual parameter name

Reply:
Done.